auction-app-backend/src/main/java/com/example/auctionapp/controller/TestController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.auctionapp.controller;
+
+import com.example.auctionapp.entity.TestEntity;
+import com.example.auctionapp.repository.TestRepository;
+import org.springframework.web.bind.annotation.*;
```

Comment:
We should avoid wildcard imports, and import only things we need. 
This can help you: https://sergiodelamo.com/blog/intellij-idea-disable-wildcard-imports.html